### PR TITLE
BY clear localstorage when logout and save modal value to it

### PR DESCRIPTION
**Pull request by Beiancaa**

### CODE REVIEW

File: app/assets/javascripts/customer_countdown_counter.js [hunk 1]
```diff
@@ -36,8 +35,64 @@ function startTimer(duration, display) {
 }
 
 window.onload = function () {
-    var fifteenMinutes = 60 * 15,
-        display = document.querySelector('#time');
-    startTimer(fifteenMinutes, display);
-
+  var fifteenMinutes = 60 * 15,
```
Comment (MwMaki) [inline app/assets/javascripts/customer_countdown_counter.js]: @Beiancaa Please use the Environment variable SIGN_IN_REWARD_TIME  to get the reward time

File: app/assets/javascripts/customer_countdown_counter.js [hunk 2]
```diff
@@ -36,8 +35,64 @@ function startTimer(duration, display) {
...
+
+
+
+
+
+
+
+
+
+
```
Comment (MwMaki) [inline app/assets/javascripts/customer_countdown_counter.js]: Please remove all empty lines

File: app/assets/javascripts/customer_countdown_counter.js (hunk 1 above)
Comment (Beiancaa) [reply app/assets/javascripts/customer_countdown_counter.js]: Yes @MwMaki  i'm still working on it..

File: config/application.yml.example [hunk 3]
```diff
@@ -22,3 +22,5 @@ COST_PER_TOKEN: '0.05'
 CONVERSION: '0.01'
 PROFIT: '0.1'
 AD_COST_FACTOR: '0.05'
+
+COUNTDOWN_TIMER: '60 * 15'
```
Comment (MwMaki) [inline config/application.yml.example]: No need for new variables we already have one

**Pull request merged**